src/main/java/core/basesyntax/model/User.java

```diff
@@ -1,10 +1,23 @@
 package core.basesyntax.model;
 
 import java.util.List;
-
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
```

Comment:
please be consistent in table names use or singular form or plural, it is common practice to use singular